sdk/storage/azure-storage-files-datalake/src/datalake_file_system_client.cpp

```diff
@@ -304,4 +304,75 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
         m_dfsUri, *m_pipeline, options.Context, protocolLayerOptions);
   }
 
+  Azure::Core::Response<Models::GetDataLakeFileSystemAccessPolicyResult>
+  DataLakeFileSystemClient::GetAccessPolicy(
+      const GetDataLakeFileSystemAccessPolicyOptions& options) const
```

Comment:
Can we call blob API for these two operations?

Reply:
We can only call blob API, there isn't such API for dfs endpoint.